packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -96,36 +96,48 @@ export default class extends EntityModel {
    * Get the org linked to the org membership.
    */
   async getOrg(graphApi: GraphApi): Promise<OrgModel> {
-    const outgoingOrgLinks = await LinkModel.getByQuery(graphApi, {
-      all: [
-        {
-          equal: [{ path: ["source", "id"] }, { parameter: this.entityId }],
-        },
-        {
-          equal: [
-            { path: ["type", "versionedUri"] },
-            {
-              parameter: SYSTEM_TYPES.linkType.ofOrg.schema.$id,
-            },
-          ],
-        },
-      ],
-    });
+    const outgoingOrgLinkEntityModels = await LinkEntityModel.getByQuery(
+      graphApi,
+      {
+        all: [
+          {
+            equal: [
+              { path: ["leftEntity", "uuid"] },
+              { parameter: this.entityUuid },
```

Comment:
See previous comment about `ownedById` :)

Reply:
Addressed in https://github.com/hashintel/hash/pull/1435/commits/7dd96c542c7645cbd835a535564da9033d838024